programs/client/Client.cpp

```diff
@@ -2264,6 +2292,7 @@ class Client : public Poco::Util::Application
             ("ask-password", "ask-password")
             ("quota_key", po::value<std::string>(), "A string to differentiate quotas when the user have keyed quotas configured on server")
             ("query_id", po::value<std::string>(), "query_id")
+            ("print_query_id", "Generate UUID query id for each query and print it")
```

Comment:
I think, we can generate query_id by default, so we don't need this config parameter.

Reply:
The behavior somehow changes when `print_query_id = true` as we always generate query_id on the client side.

Reply:
That's fine if client will always generate query_id.